docker-vlogger/Dockerfile

```diff
@@ -16,6 +16,7 @@
 
 ARG ALPINE_VERSION
 ARG BASE_IMG
+ARG TARGETARCH
```

Comment:
This isn't used in current dockerfile, and I cannot find where we passed it in.

Reply:
This is an implicit Dockerfile arg that the Build system automatically recognizes: https://docs.docker.com/reference/dockerfile/#automatic-platform-args-in-the-global-scope

Reply:
You don't used it in this file, do we probably can remove it in this file?